src/locales/ru.json

```diff
@@ -0,0 +1,129 @@
+{
+	"common": {
+		"LoadingImages": "Загрузка образов",
+		"En": "Английский",
+		"Common": "Общие",
+		"package_0": "{{count}} пакет",
+		"package_1": "{{count}} пакета",
+		"package_2": "{{count}} пакетов",
+		"Packages": "Пакеты",
+		"Details": "Детали",
+		"Ru": "Русский",
+		"Download": "Загрузить",
+		"Edit": "Добавить заметку",
+		"From": "Из",
+		"ClickHere": "Нажмите здесь",
+		"Delete": "Удалить",
+		"Terminal": "Тест драйв",
+		"Rebuild": "Пересобрать билд",
+		"ErrorLog": "Отчет о ошибках",
+		"Dashboard": "Главная",
+		"Blog": "Блог",
+		"Docs": "Документация",
+		"Feedback": "Обратная связь",
+		"selected": "выбрано",
+		"select": "выбрать",
+		"and": "и"
+	},
+	"bytes": {
+		"B": "б",
+		"EB": "Эб",
+		"GB": "Гб",
+		"kB": "Кб",
+		"MB": "Мб",
+		"PB": "Пб",
+		"TB": "Тб",
+		"YB": "Йб",
+		"ZB": "Зб"
+	},
+	"Packages": {
+		"Base": "Базовые пакеты",
+		"Dependencies": "Зависимости",
+		"Added": "Добавленные пакеты",
+		"PerPage": "На странице",
+		"PackageName": "Имя пакета",
+		"Description": "Описание",
+		"Version": "Версия",
+		"Size": "Размер"
+	},
+	"Image": {
+		"StartedAt": "Дата старта",
+		"CreatedAt": "Дата создания"
+	},
+	"Dashboard": {
```

Comment:
why you took i18n filled files? you have a many unnecessary files from cusdeb. clean your branch before give it on review

Reply:
That pull request is in 'Draft' state and review was not required by me. Eugene ignited this

Reply:
in first, initial commit remove unused i18 files